src/MongoDB/Query.c

```diff
@@ -277,8 +295,8 @@ static bool php_phongo_query_init(php_phongo_query_t* intern, zval* filter, zval
 	PHONGO_QUERY_OPT_BOOL("exhaust", options, "exhaust");
 	PHONGO_QUERY_OPT_DOCUMENT("max", options, "max")
 	else PHONGO_QUERY_OPT_DOCUMENT("max", modifiers, "$max");
-	PHONGO_QUERY_OPT_INT64("maxScan", options, "maxScan")
-	else PHONGO_QUERY_OPT_INT64("maxScan", modifiers, "$maxScan");
+	PHONGO_QUERY_OPT_INT64_DEPRECATED("maxScan", options, "maxScan")
+	else PHONGO_QUERY_OPT_INT64_DEPRECATED("maxScan", modifiers, "$maxScan");
```

Comment:
AFAIK, `modifiers` is deprecated ([DRIVERS-347](https://jira.mongodb.org/browse/DRIVERS-347)). Perhaps we should simply log a notice when _that_ option is used. If so, that should be its own PHPC ticket. I think we can still do it for 1.5.0; however, I'd then suggest we raise that notice once for `modifiers` and forego additional notices for sub-options like `$maxScan`.

Thoughts?

Reply:
It should still throw a notice for ``$maxScan`` and ``snapshot``, as they're still deprecated options on their own. Otherwise people change the modifiers to options (and ``$maxScan`` to ``maxScan``), and they would *then* get a notice. Better to see them all in one go.